app/controllers/api/tags_controller.rb

```diff
@@ -0,0 +1,87 @@
+module Api
+  # Api controller for tags
+  # Uses Rails' RESTful routes (check 'rake routes' for the configured routes)
+  class TagsController < MainApiController
+    DEFAULT_FIELDS = [:id, :name, :description, :creator, :use].freeze
+
+    def index
+      @assignment = Assignment.find_by(id: params[:assignment_id])
```

Comment:
For the API routes you generally don't need to use instance variables because you aren't going to use these variables in view files. Here you should use `assignment` instead of `@assignment`.